lib/puppet/parser/functions/consul_sorted_json.rb

```diff
@@ -6,11 +6,13 @@ class << self
 
     def sorted_generate(obj)
       case obj
-        when Fixnum, Float, TrueClass, FalseClass, NilClass
-          return obj.to_json
+        when NilClass
+          return "null"
```

Comment:
Well, yes when you print it, it becomes a string, yes.

I understand that this function actually generates "content" for puppet, so maybe returning the string is ok. 

Ug, I just don't like this function :(

Don't forget to adjust the "non"-pretty function as well.

Please add a test for this, I don't want to see this regress. 

Something like:

    it 'handles null correctly' do
      expect(subject.call({"foo" => null}).to eql('"foo"=>null')
    end